agent/lib/kontena/fluentd/server.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'server_connection'
+require_relative 'message_processor'
+
+module Kontena
+  module Fluentd
+    class Server
+      include Celluloid
+      include Kontena::Logging
+
+      finalizer :finalize
+
+      def initialize(autostart = true)
+        info 'initialized'
+        @connections = []
+        @queue = SizedQueue.new(1000)
+        @message_processor = Kontena::Fluentd::MessageProcessor.new(@queue)
```

Comment:
Separate supervised actor, or this will leave the old message processors actors hanging around if this actor crashes and restarts, spawning a new actor?